app/controllers/home_controller.rb

```diff
@@ -10,6 +10,7 @@ def feed
   def how_government_works
     sitewide_setting = load_reshuffle_setting
     @is_during_reshuffle = sitewide_setting.on if sitewide_setting
+    @prime_minister = MinisterialRole.where(slug: 'prime-minister').first.current_person
```

Comment:
`MinisterialRole.find_by(slug: 'prime-minister').current_person` would be slightly better perhaps, to make it clear that the slug is unique.